src/dialog.tsx

```diff
@@ -213,9 +213,16 @@ const NativeModalDialog = ({
         message,
         title,
     });
+    const isOpenRef = React.useRef(false);
 
     React.useEffect(() => {
-        showNativeDialog(paramsRef.current);
+        if (isOpenRef.current) {
```

Comment:
Maybe you can add a comment here explaining why is this needed

Reply:
ok, added